src/classes/CON_Phone.cls-meta.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<ApexClass xmlns="urn:metadata.tooling.soap.sforce.com" fqn="CON_Phone">
```

Comment:
wow, i've never even seen this before...

Reply:
I've seen it if I create a new Apex class in VS Code instead of creating it in Salesforce first and pulling it down via the metadata API, I think. I'm assuming that's what happened here.